pom.xml

```diff
@@ -93,16 +93,14 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
       <version>${junit.version}</version>
```

Comment:
I think this becomes <scope>provided</scope> so that code using this library is forced to supply it's own implemenation.